neoq.go

```diff
@@ -238,8 +238,8 @@ func Backend(backend Neoq) ConfigOption {
 	}
 }
 
-// ConnectionString configures neoq to use connection string for backend initialization
-func ConnectionString(connectionString string) ConfigOption {
```

Comment:
I don't think this one should live on the wider interface, but instead in the backends directly. Build a postgres backend using this connection string. Same for redis. Memory..  not needed. SQS here's the authentication scheme. Etc.

Reply:
That makes sense, I'll kill it.